Editor/AGS.Editor/BuildTargets/BuildTargetWindows.cs

```diff
@@ -191,15 +191,15 @@ public void UpdateWindowsEXE(string filename, CompileMessages errors)
         public override bool Build(CompileMessages errors, bool forceRebuild)
         {
             if (!base.Build(errors, forceRebuild)) return false;
-            string compiledDir = AGSEditor.OUTPUT_DIRECTORY;
+            string compiledDataDir = BuildTargetsInfo.FindBuildTargetByName(BuildTargetsInfo.DATAFILE_TARGET_NAME).GetCompiledPath();
```

Comment:
Missed this one as well, sorry. Did a search against `IBuildTarget` and forgot these one-liners.